config/rbac/clusterapplicationresourcemapping_editor_role.yaml

```diff
@@ -0,0 +1,24 @@
+# permissions for end users to edit clusterapplicationresourcemappings.
```

Comment:
is this file used?

Reply:
The RBAC-related files are auto-generated by kubebuilder. Probably helpful for those who are packing this project (say Helm charts).

Reply:
ok. I was a bit surprised to not see it listed in the kustomization.yaml file, even if commented